warehouse/stats.go

```diff
@@ -152,7 +154,10 @@ func (job *UploadJobT) generateUploadAbortedMetrics() {
 
 	// Set the upload_aborted stat
 	attempts := job.getAttemptNumber()
-	tags := []Tag{{Name: "attempt_number", Value: strconv.Itoa(attempts)}}
+	tags := []Tag{
+		{Name: "attempt_number", Value: strconv.Itoa(attempts)},
+		{Name: "error_mapping", Value: string(model.Noop)},
```

Comment:
should value be `model.Noop` here ?

Reply:
Thanks @abhimanyubabbar. upload_aborted metrics is being sent twice at https://github.com/rudderlabs/rudder-server/blob/dfc227302b23bf8ceb3cdc72601f1bbd94435178/warehouse/upload.go#L1747 and https://github.com/rudderlabs/rudder-server/blob/dfc227302b23bf8ceb3cdc72601f1bbd94435178/warehouse/stats.go#L135

Reply:
Removing it from here.